### PR TITLE
Bootstrap minion no longer required to have admin role

### PR DESCRIPTION
**Pull request by ricardoasmarques**
If https://github.com/ceph/ceph-salt/pull/383 is merged, bootstrap minion will no longer be required to have the admin role.

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

**Review by lproven — APPROVED** [review]

**Review by asettle — APPROVED** [review]

**Comment by asettle** [comment]
Applying do not merge label until ceph/ceph PR is merged. Thanks @ricardoasmarques

**Comment by ricardoasmarques** [comment]
https://github.com/ceph/ceph-salt/pull/383 has been merged

**Pull request merged**
